tools/devtool

```diff
@@ -1118,6 +1121,19 @@ cmd_tag() {
     say "Tag v$version created."
 }
 
+# Runs a continuous integration test by running these devtool subcommands:
+# 1. devtool -y test
+# 2. devtool -y build --release
+# 3. devtool strip
+cmd_ci() {
```

Comment:
This is to use in our CI pipelines?

Reply:
Yep, we should test that a PR also doesn't break the release steps, whatever those steps are.